kong/reports.lua

```diff
@@ -121,25 +121,23 @@ local function send_report(signal_type, t, host, port)
     end
   end
 
-  local sock = udp_sock()
-  local ok, err = sock:setpeername(host, port)
+  local sock = tcp_sock()
+  sock:settimeout(1000)
```

Comment:
1 second seems to short of a timeout. The network could be congested for any reason, and we might lose some reports given no the lack of retry policy. There should be no harm in increasing this value to a much higher timeout given no client or user is waiting for a response here.

Besides, better use the newer `settimeouts` method instead.